hack/gendocs/main.go

```diff
@@ -5,12 +5,16 @@ package main
 
 import (
 	"log"
+	"os"
 
 	oms "github.com/codesphere-cloud/oms/cli/cmd"
 	"github.com/spf13/cobra/doc"
 )
 
 func main() {
+	// to ensure the stable binary name is used and not the temporary path
+	os.Args[0] = "oms"
```

Comment:
Not sure it's a good idea to override it here tbh. Also, I think it should be `oms-cli` since that's what it's currently built as.